tools/gen-jit-template.py

```diff
@@ -110,23 +110,18 @@ def parse_argv(EXT_LIST, SKIP_LIST):
     for ext in EXT_LIST:
         SKIP_LIST += INSN[ext]
 
-
-def remove_comment(str):
-    str = re.sub(r'//[\s|\S]+?\n', "", str)
-    return re.sub(r'/\*[\s|\S]+?\*/\n', "", str)
-
-
 # parse_argv(EXT_LIST, SKIP_LIST)
 # prepare PROLOGUE
 output = ""
 f = open('src/rv32_template.c', 'r')
 lines = f.read()
+# remove_comment
+lines = re.sub(r'/\*[\s|\S]+?\*/', "", lines)
```

Comment:
Is it strong enough to resolve #296 ?

Reply:
Yes.

Reply:
Append `Close #296` at the end of git commit messages and improve the comments of `src/rv32_template.c` as a complete example starting with `RVOP`.